### ISSUE TITLE
Library API Reference in the Wiki

### ISSUE THREAD

**Issue by thejonan** [dotnet-state-machine/stateless#436]
Thanks for the project and your efforts!

While examples are a nice way to get a feeling about the functionality of a certain library, the API reference are the "ultimate source of truth" - I really miss it for Stateless. Given that you have methods documented, isn't it possible (and easy) to make an automated action, which publishes a reference to all classes and methods, to the repo-'s Wiki?

It would be great!

**Comment by HenningNT** [comment]
That's a an excellent idea!
I've never set up such a think, do you know how this is done?

**Comment by thejonan** [comment]
Yes, I do. I haven't done it in this _particular_ setup, but that shouldn't be a problem. I'll try to prepare a PR in the next few days.

**Comment by HenningNT** [comment]
Oh, that would be lovely!
